examples/cuda_umpire.cpp

```diff
@@ -0,0 +1,118 @@
+//#if defined(MFEM_USE_UMPIRE) && (defined(MFEM_USE_CUDA) || defined(MFEM_USE_HIP))
```

Comment:
This is failing Travis now: https://travis-ci.com/github/mfem/mfem/jobs/471197181#L818. Should it be guarded?

Also: please add the Copyright header :)

Reply:
I was going to say this is not the place for this, but I just noticed https://github.com/mfem/mfem/pull/1644#issuecomment-759148473 :)

Please move to `tests/`.

Reply:
I'll move it once the interface is set, I had some trouble running it quickly from the test system, is there a way to quickly run 1 test once if it's there?